restapi/src/main/java/io/stargate/web/docsapi/dao/DocumentDB.java

```diff
@@ -322,6 +333,29 @@ public ResultSet executeSelect(
         .join();
   }
 
+  public ResultSet executeSelect(
+      String keyspace,
+      String collection,
+      List<BuiltCondition> predicates,
+      int pageSize,
+      ByteBuffer pageState)
+      throws UnauthorizedException {
+    // Run generic authorizeDataRead for now
```

Comment:
could you elaborate more on that comment?

Reply:
For now we'll just use authz that tells us whether the user has access to the table but in the future we should add a more granular authz method that's able to work with documents.